plugins/auth/index.js

```diff
@@ -113,15 +113,30 @@ exports.register = (server, options, next) => {
                 accessTokenName: 'access_key',
                 allowCookieToken: false,
                 allowQueryToken: true,
-                validateFunc: (token, cb) => {
-                    if (token !== pluginOptions.temporaryAccessKey) {
-                        return cb(null, false, {});
+                validateFunc: function _validateFunc(token, cb) {
```

Comment:
What's different between `this.server` and `server` here? https://github.com/screwdriver-cd/screwdriver/blob/cf158a135aa11171683d9d4bc8cc58fc2e596f9b/plugins/auth/index.js#L112